packages/xs-vat-worker/xs-lin.mk

```diff
@@ -3,6 +3,19 @@ TOOLS=$(MODDABLE)/build/bin/lin/release/
 NODE_MODULES=$(PWD)/node_modules
 ROOT=$(PWD)/../..
 
+.PHONY: all
+
+all:
+	$(MAKE) -f xs-lin.mk build
```

Comment:
why `$(MAKE)` in a subprocess rather than just make `all` depend on `build`?

Reply:
I wanted a place to emit distinctive `echo` lines just before each step ran (because they were failing as I figured things out); using the normal `make` dependencies didn't make it clear when the stages were happening. That said, now that it works, I don't strictly need it anymore.

Reply:
no strong feelings.